content/en/docs/bmf/multiple_features/ffmpeg_fully_compatible/_index.md

```diff
@@ -90,6 +90,8 @@ In order to parallel modules, the encode module will be apply in scheduler 1 whi
         )
 ```
 
+BMF supports ffmpeg GPU codec as well. E.g. you can set `"hwaccel": "cuda"` and `"codec": "hevc_nvenc"` to use GPU for decoding and encoding. Check out the [gpu transcoding section](../gpu_hardware_acc/gpu_transcoding/_index.md) for more details.
```

Comment:
Could we change the page jump style to the real link in the website? 
eg: http://babitmf.github.io/docs/bmf/multiple_features/gpu_hardware_acc/gpu_transcoding ,